namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: sentence-planning-preprod
+spec:
+  limits:
+  - default:
+      cpu: 250m
+      memory: 500Mi
+    defaultRequest:
+      cpu: 125m
```

Comment:
please reduce your cpu default request to 10m. This is not a limit, it's just the amount of CPU the cluster will reserve for each of your containers.

Reply:
I have made this change to our dev and preprop environments